mozdownload/scraper.py

```diff
@@ -265,6 +311,7 @@ def total_seconds(td):
 
         self.logger.info('Downloading from: %s' % self.url)
         self.logger.info('Saving as: %s' % self.filename)
+        self.logger.info('Directory: %s' % os.path.split(self.url)[0])
```

Comment:
I wouldn't necessarily modify download for the hash. Instead we might have to think about to pass in the `url` to download and do not automatically assume its `self.url`. With that we could re-use it for the real download and the checksum file.